src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Data/Migrations/20200818132003_IdentityServer4.Designer.cs

```diff
@@ -0,0 +1,405 @@
+﻿// <auto-generated />
```

Comment:
autogenerated by EF migrations

Reply:
Can we run the migrations during the build instead of checking in the generated files❔ Or, is that just too much of a pain❔

Reply:
That's painful, specially for a sample.